Update sentry-sdk to version 1.34.0 and enable django extras

ttys0dev opened the pull request:
Probably a good idea to add the django extras(I don't think that option was available in the previous version).

mlissner commented:
Seems fine, but do we need the Celery and Redis extras too or are those built in? Can you link to the release notes too, please?

ttys0dev commented:
Rebased and updated to 1.34.0.

> Seems fine, but do we need the Celery and Redis extras too or are those built in?

Added celery extras. Redis extras/integration is apparently builtin with 1.34.0.

> Can you link to the release notes too, please?

https://github.com/getsentry/sentry-python/blob/master/CHANGELOG.md

mlissner commented:
Cool. Rebased and set for auto-merge.
